x-pack/legacy/plugins/alerting/server/routes/get.test.ts

```diff
@@ -17,6 +17,8 @@ const mockedAlert = {
   params: {
     bar: true,
   },
+  createdAt: new Date(),
+  updatedAt: null,
```

Comment:
I would think the normal case would be that `updatedAt` would never be null, so seems like it should be set to a non-null value here.